scripts/local_cloudbuild_test.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python3
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit test for local_cloudbuild.py"""
+
+import argparse
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+import unittest
+import unittest.mock
+
+import yaml
+
+import local_cloudbuild
+
+
+class ValidationUtilsTest(unittest.TestCase):
```

Comment:
This makes me pretty sad, as I'm very much pushing for us to use py.test with our open-source stuff as the tests tend to be much more readable, easier to maintain, and much more approachable for outsiders.

However, this is a one-off utility script so I won't fight you too much on it.